src/mdtest.c

```diff
@@ -682,9 +700,16 @@ void mdtest_read(int random, int dirs, const long dir_iter, char *path) {
 
         /* read file */
         if (read_bytes > 0) {
+            read_buffer[0] = 42;
```

Comment:
What is 42?  Can we make it an enum typedef or something?

Reply:
It is just to overwrite the read buffer with something that is not in the pattern. It just cannot be 0.